int8                    object_count
UltraSonicTarget[16]    us_objects